react/features/e2ee/components/E2EESection.js

```diff
@@ -149,12 +161,25 @@ class E2EESection extends Component<Props, State> {
  * @returns {Props}
  */
 function mapStateToProps(state) {
-    const { enabled } = state['features/e2ee'];
+    const { enabled: e2eeEnabled, maxMode } = state['features/e2ee'];
     const { e2eeLabels } = state['features/base/config'];
 
+    let descriptionResource = '';
```

Comment:
Add a comment and leave it undefined if e2eelabels.description is set, so we know it's not used.